frontend/app/components/encryptable/form.js

```diff
@@ -1,5 +1,7 @@
 import { action } from "@ember/object";
-import AccountValidations from "../../validations/encryptable";
+import EncryptableValidations from "../../validations/encryptable";
```

Comment:
completely focus on backend for now.
make sure all backend functionality is working ... then implement frontend.
TDD for the win